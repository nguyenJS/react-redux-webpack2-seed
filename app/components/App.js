import React, { Component } from 'react';
import List from './list/List';

export default class App extends Component {
  render() {
    return (
    <div>
      <h2 id="heading">Premier League Teams</h2>
      <List />
    </div>
    );
  }
}

