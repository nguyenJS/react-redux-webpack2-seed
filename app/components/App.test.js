import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App Component', () => {
  it('should render the component', () => {
    // when
    const wrapper = shallow(
        <App />
    );
    // then
    expect(wrapper.root.node).toMatchSnapshot();
  });
});

