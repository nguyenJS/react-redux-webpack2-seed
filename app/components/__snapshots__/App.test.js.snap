// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render the component 1`] = `
<div>
  <h2
    id="heading"
  >
    Premier League Teams
  </h2>
  <List />
</div>
`;
